component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -1988,7 +1994,8 @@ private ComponentFamilyMeta getOrCreateComponent(final String component) {
                             iconFinder.findIcon(familyAnnotationElement), comp,
                             Class.class.isInstance(familyAnnotationElement)
                                     ? getPackage(Class.class.cast(familyAnnotationElement))
-                                    : ""))
+                                    : "",
```

Comment:
same comment about the contstant NONE, where it's applicable

Reply:
not same parameter, it's packagename